contrib/babelfishpg_common/src/collation.c

```diff
@@ -1205,20 +1233,20 @@ BABELFISH_CLUSTER_COLLATION_OID()
 {
 	if (sql_dialect == SQL_DIALECT_TSQL)
 	{
-		get_server_collation_oid_internal(false);	/* set and cache
+		Oid db_coll = get_database_or_server_collation_oid_internal(false);	/* set and cache
 													 * server_collation_oid */
 
-		if (OidIsValid(server_collation_oid))
-			return server_collation_oid;
+		if (OidIsValid(db_coll))
+			return db_coll;
```

Comment:
Redundant check, missingOk is false in `get_database_or_server_collation_oid_internal`